src/postgres/ClientConfigClass.ts

```diff
@@ -0,0 +1,41 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { ClientConfig } from 'pg';
+
+export class ClientConfigClass implements ClientConfig {
```

Comment:
I think this class is overkill. It appears it's only _really_ used in `PostgreSQLDatabaseTreeItem` here:
```
public async connectToDb(): Promise<Schema[]> {
        const accountConnection = await connectToPostgresClient(this.clientConfig);
        const db = await pgStructure(accountConnection);
        return db.schemas;
    }
```

But we would normally create a simple object and pass it in like this:
```
public async connectToDb(): Promise<Schema[]> {
        const accountConnection = await connectToPostgresClient({
            host: this.host,
            database: this.databaseName,
            // etc.
        });
        const db = await pgStructure(accountConnection);
        return db.schemas;
    }
```